src/ens.js

```diff
@@ -556,7 +500,21 @@ export class ENS {
     let namehash = getNamehash(reverseNode)
     return Resolver.setName(namehash, name)
   }
-
+  async wildcardResolverDomain(name){
+    let res, resNew
+    const provider = await getProvider()
+    const resolverAddress = await this.getResolver(name)
+    const _resolverAddress = await provider._getResolver(name)
```

Comment:
??? what is going on here?

Reply:
_getResolvere calls registry.resolver directly bypassing wildcard. I initially added it to address
https://github.com/ensdomains/ens-app/pull/1468#discussion_r854544730  (`We should show that message for any domain which a) doesn't exist in the registry, or b) has a resolver that implements resolve`) but if I think about it we should just check if the resolver implements `resolve` (my original implementation).